projects/admin/src/app/acquisition/components/order/order-detail-view/order-detail-view.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+  
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+  
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div *ngIf="record$ | async as record" class="container">
+  <h1 class="mb-3">{{ record.metadata.order_number }}</h1>
+  <section class="my-3 row">
+    <!-- ORDER DETAIL ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
```

Comment:
```suggestion
    <!-- ORDER DETAIL (:~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
```
Oh, a jellyfish ! :wink: 
Never mind, it just reminds me https://fr.wikipedia.org/wiki/Telecomix#M%C3%A9duse